lib/ipam/ipam_test.go

```diff
@@ -446,6 +456,64 @@ var _ = testutils.E2eDatastoreDescribe("IPAM tests", testutils.DatastoreAll, fun
 		})
 	})
 
+	Describe("Reservation tests", func() {
```

Comment:
We need to add a test case where an IP is already been reserved in a block? For example, windows nodes reserves 4 IPs for a block?